vendor/assets/stylesheets/dvl/core/forms.scss

```diff
@@ -408,19 +409,23 @@ form {
   margin-bottom: $rhythm * 2;
 }
 
-.form_item_horiz_label {
+.form_item_horiz_label,
+.form_item_horiz legend {
```

Comment:
I wanted to avoid having a single class apply to both a an element's container and the element itself. That gets difficult to maintain quickly. We could wrap every `legend` in a `div.form_item_horiz_label`, but that just seemed laborious to do manually every time.